src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -11,9 +13,8 @@ import no.skatteetaten.aurora.boober.service.CantusService
 import no.skatteetaten.aurora.boober.utils.boolean
 import no.skatteetaten.aurora.boober.utils.isListOrEmpty
 import no.skatteetaten.aurora.boober.utils.isValidDns
+import no.skatteetaten.aurora.boober.utils.startsWithSlash
 import no.skatteetaten.aurora.boober.utils.validUrl
-import org.springframework.beans.factory.annotation.Value
-import org.springframework.stereotype.Service
 
 @Service
 class AzureFeature(
```

Comment:
Jeg skjønte ikke hvorfor vi skal koble sammen forskjellige azure-ting, men det er kanskje en grunn. Kunne det stått i javadoc? Tenker api-management og AAD egentlig ikke har så mye til felles, selv om begge deler er i Azure..

Reply:
Tanken var at det skulle være logisk, altså at dette er config som har med azure å gjøre og derfor henger det sammen. 

- [ ] Tilføy overordnet javadoc for AzureFeature